lib/errors.coffee

```diff
@@ -17,10 +17,21 @@ limitations under the License.
 chalk = require('chalk')
 errors = require('resin-cli-errors')
 patterns = require('./utils/patterns')
-Raven = require('raven')
 Promise = require('bluebird')
-
-captureException = Promise.promisify(Raven.captureException.bind(Raven))
+analytics = require('analytics.node').core
+Raven = require('analytics.node').sentryIntegration
+dsn = require('./config').sentryDsn
+release = require('../package.json').version
+ravenOptions =
+	config: dsn
+	release: release
+	captureUnhandledRejections: true
+	disableConsoleAlerts: true
+analytics.addIntegration(Raven)
+analytics.initialize 'Sentry': ravenOptions
+analytics.setContext analytics.anonymize(extra:
+	args: process.argv
+	node_version: process.version)
```

Comment:
Why do we need to duplicate this between `app.coffee` and here? Does this only configure the `analytics` instance, not globally? That's slightly surprising, but I guess it makes sense.

It'd be good to put these in a separate file that's imported into both though, rather than duplicating it identically.

Reply:
I've removed those from app.coffee